CONTRIBUTING.md

```diff
@@ -13,6 +13,10 @@ Some people might also use the [_fastlane_ tag on StackOverflow](https://stackov
 - To start working on _fastlane_, check out [YourFirstPR.md][firstpr]
 - For some more advanced tooling and debugging tips, check out [ToolsAndDebugging.md](ToolsAndDebugging.md)
 
+### Google CLA
```

Comment:
Let's expand this out to "Google Contributor License Agreement (CLA)" for people who've never encountered this acronym before 👍